tools/amd_build/build_amd.py

```diff
@@ -4,15 +4,11 @@
 import argparse
 import os
 import sys
+from pathlib import Path
 
 
-sys.path.append(
-    os.path.realpath(
-        os.path.join(
-            __file__, os.path.pardir, os.path.pardir, os.path.pardir, "torch", "utils"
-        )
-    )
-)
+REPO_ROOT = Path(__file__).absolute().parents[2]
+sys.path.append(str((REPO_ROOT / "torch" / "utils").resolve()))
```

Comment:
I vaguely remember talking about it in the past: it will resolve to a different folder if `__file__` is a symlink to some other place. I.e. imagine a folder structure like (which is used by build system that bazel/buck, but could be the case of many other builds):
```
   - actual_files
           - build_amd.py
           - hipify.py
   - deploy 
     - tools
          - amd_build
              - build_amd.py -> ../../../build_amd.py   
     - torch
         -  utils
            - hipify.py -> ../../../hipify.py 
```
In the previous version of the logic, it will resolve path correctly, but after the reorder it will point to non-existing `/torch/utils/` folder